File: packages/android-native/app/src/main/java/dev/tatliving/palmvellum/organizers/data/sync/SupabaseConfig.kt
package dev.tatliving.palmvellum.organizers.data.sync

/**
 * Supabase project + publishable (frontend-safe) key. Same project as the
 * PWA. The publishable key is meant to live in clients; RLS protects data.
 */
object SupabaseConfig {
    const val URL = "https://jrkwncplngmznfzzqwee.supabase.co"
    const val PUBLISHABLE_KEY = "sb_publishable_UoFQ7p6EPTm0cbqimURGPQ_J1HO_aR-"
}

File: packages/mac-native/Sources/PalmKit/SupabaseConfig.swift
import Foundation

/// Supabase project + publishable (frontend-safe) key. SAME project as the PWA
/// and the Android app — see `android-native/.../SupabaseConfig.kt`. The
/// publishable key is meant to ship in clients; row-level security protects data.
public enum SupabaseConfig {
    public static let url = URL(string: "https://jrkwncplngmznfzzqwee.supabase.co")!
    public static let publishableKey = "sb_publishable_UoFQ7p6EPTm0cbqimURGPQ_J1HO_aR-"

    /// Where users manage BYOK AI keys / credits (the PWA settings page).
    public static let webSettingsURL = "https://tatliving.dev/palmvellum/app/settings"
}
